src/termio/Exec.zig

```diff
@@ -610,6 +641,7 @@ pub const ThreadData = struct {
     // The preallocation size for the write request pool. This should be big
     // enough to satisfy most write requests. It must be a power of 2.
     const WRITE_REQ_PREALLOC = std.math.pow(usize, 2, 5);
+    const FlatpakCompletion = if (Subprocess.FlatpakHostCommand != void) Subprocess.FlatpakHostCommand.Completion else struct {};
```

Comment:
Not sure if an empty struct is the best bet here, but I don't know if there's anything that can initialize void or struct.